Modified IRCBot | Fixes #1, #4, 2/4 of #2

darthbhyrava opened the pull request:
- Standard golang structure implemented for ease of scalability and future functionality implementations. Read more here: https://golang.org/doc/code.html
- Added functionality for following commands
  1)!hi (changed to !rule), 
  2)!pastebin
  3)!stories

Fixes issues #1 #4 and parts of #2

amoghbl1 committed:
pong request end, and additions

amoghbl1 committed:
minor changes

darthbhyrava committed:
Copy of @amoghbl1's gobot | commented doubts

//Answer the comments with a new line of comments.

amoghbl1 committed:
teehee

darthbhyrava committed:
Merge pull request #1 from amoghbl1/haha

Haha

darthbhyrava committed:
initial delete

darthbhyrava committed:
second delete

darthbhyrava committed:
Second commit

darthbhyrava committed:
Updated with Installation instructions

darthbhyrava committed:
DNS Response error fix

darthbhyrava committed:
Deleted old "initial comments and queries" file

mrinaldhar commented on README.md:
```diff
@@ -1,3 +1,10 @@
 ## This bot is currently going to function on #osdg-iiith on freenode.
 This is a small irc bot written in golang, you're welcome to add functionality to it!
+
+Instructions:
```
I feel like these instructions could be better phrased. Example: "set path to /bin and then execute the 'bot' executable". There's a better way to say that.

darthbhyrava replied:
# Simple irc bot written in golang

## This bot is currently going to function on #osdg-iiith on freenode.

This is a small irc bot written in golang, you're welcome to add functionality to it!

Instructions:
- Install golang: https://golang.org/doc/install
- Setup the GOPATH environment variables: https://golang.org/doc/code.html
- Build and install main and functions packages using the 'go tool': https://golang.org/doc/code.html
- Command Executables are generated in /bin. PATH variable can be reset to execute them.
- If server connection problems arise, check DNS nameserver address in /etc/resolv.conf

//Is this better?

mrinaldhar commented on goIRCBot/src/functions/functions.go:
```diff
@@ -0,0 +1,53 @@
+	val3:="Blog : http://iiitosdg.wordpress.com/"
+	val4:="IRC : Well, you guys are already here aren't you :P"
+	val5:="GitHub : https://github.com/OSDG-IIITH/"
+	val6:="Want to get a project forked under the github group? Register it at http://osdg.iiit.ac.in/github/"
```
That URL doesn't exist ATM. Let's add an issue for this on osdg-www repository.

darthbhyrava replied:
Removed the URL.

mrinaldhar commented on goIRCBot/src/functions/functions.go:
```diff
@@ -0,0 +1,53 @@
+	val4:="IRC : Well, you guys are already here aren't you :P"
+	val5:="GitHub : https://github.com/OSDG-IIITH/"
+	val6:="Want to get a project forked under the github group? Register it at http://osdg.iiit.ac.in/github/"
+	val7:="Doing GSoC this summer? Check out http://osdg.iiit.ac.in/gsoc15/"
```
Why does it point to that page with the text "Doing GSoC this summer?" ?

I think a better way to introduce that page could be "People who did GSoC last summer: " or something else that sounds better than that example :P

darthbhyrava replied:
Edited. :P

darthbhyrava committed:
Edits for notes 2,3 | Mrinal

Removed link to non-existent github repo | Edited link description for last summer's GSoC list

darthbhyrava committed:
Readme instructions edited for better readability
